packages/mds-compliance-api/spec/flat-spec.json

```diff
@@ -193,7 +193,7 @@
             ]
           }
         ],
-        "summary": "Get compliance snapshot_ids",
+        "summary": "Get compliance snapshot_ids given a token. Sometimes, an opaque token will be provided as part of a URI to point to a collection of snapshots, this endpoint can be utilized to access those snapshots.",
```

Comment:
Sorry, but what does "opaque" mean here? What's an opaque token, and conversely, what's a transparent token?